tests/PactContinuationSpec.hs

```diff
@@ -1186,20 +1186,17 @@ twoPartyEscrow testCmds act = do
   allResults <- runAll allCmds
 
   runResults allResults $ do
-    sysModuleCmd `succeedsWith` textVal "system module loaded"
-    acctModuleCmd `succeedsWith` textVal "TableCreated"
-    testModuleCmd `succeedsWith` textVal "test module loaded"
-    createAcctCmd `succeedsWith`  Nothing -- Alice should be funded with $100
-    resetTimeCmd `succeedsWith`  Nothing
-    runEscrowCmd `succeedsWith`  Nothing
-    balanceCmd `succeedsWith` decValue 98.00
+    sysModuleCmd `succeedsWith` (`shouldBe` textVal "system module loaded")
+    acctModuleCmd `succeedsWith` (`shouldBe` textVal "TableCreated")
+    testModuleCmd `succeedsWith` (`shouldBe` textVal "test module loaded")
+    succeeds createAcctCmd -- Alice should be funded with $100
```

Comment:
Does the comment provide any value? (I've noticed the removal [here](https://github.com/kadena-io/pact/pull/1222/files#diff-92311cca589a9cd01bc5b9b5efd95c8953e4bd934005d44134c0731254883283L154) too).